frontend/src/pages/NewRun.test.tsx

```diff
@@ -1699,7 +1699,7 @@ describe('NewRun', () => {
       await screen.findByDisplayValue('prefilled value 1');
       await screen.findByLabelText('param-2');
       await screen.findByDisplayValue('prefilled value 2');
-    });
+    }, 10000);
```

Comment:
Can I ask what this change is used for?

Reply:
Explained in PR description https://github.com/kubeflow/pipelines/pull/9951#issue-1877927798:

```
Extended timeout for NewRun test since jest has been upgraded implicitly to 27. It is the version which modern fake timer has become default.
```

I think a better fix is to explicitly pin the jest version, and add a jest config file for the fake timer. Since some effort needs to be invested for that, I want to narrow the PR focus here. (This workaround is for passing the presubmit test)